### DISCUSSION TITLE
Site internet

### DISCUSSION THREAD

**Discussion by chrabou** [lucile98-ode/GPI-L2BHA-FENOUIL#29] [Problèmes]
Il faudrait voir pour faire la liaison entre le site et l'application en rapport avec la machine virtuelle et ce qu'on peut en faire réellement.
@Avicenne-b

**Comment by lucile98-ode** [comment]
Il n'est pas nécessaire d'avoir une machine virtuelle @Avicenne-b  (solution apportée en interne par @amrx12 )

**Comment by amrx12** [comment]
**Oui normalement la machine virtuelle se génère automatiquement lors du lancement du code JAVA. Lorsque le programme est compilé pour afficher l'interface graphique.**
